pp3/module/Application/src/Application/Controller/PluginController.php

```diff
@@ -198,6 +192,12 @@ public function confirmAction() {
                         $plugin->addCategory($cat2);
                     }
                 }
+                // save image
```

Comment:
This has been only moved after entity gets persisted so $plugin->getId() return correct ID